### DISCUSSION TITLE
New Dev & Staging

### DISCUSSION THREAD

**Discussion by luketchang** [nomad-xyz/monorepo#223] [Announcements]
### Development
 - [x] Deploy agents to full development setup at [db607dd](https://github.com/nomad-xyz/rust/pull/110/commits/db607ddeb34cf9755b86d86606fd0dfa9fa9865a) (@arnaud036)
 - [x] Bump TS packages and SDK to use configuration `0.1.0-rc.12` (@Imti)
 - [x] Run indexer against dev
   - [x] Bump indexer version with new SDK (@ltchang2019)
   - [x] Redeploy new instance indexer against dev (@arnaud036)
   - [ ] Fix dev indexer (@kekonen)
   - `Error: 1111: Left roots for replica 2221 supposed to be 0, but is 2 replica for domain 2221`
 - [x] Deploy dev GUI (@ErinHales)
 - [x] QA token transfer E2E (@ltchang2019 )
    - [x] check sending tokens 
    - [x] check that tokens are received 
- [x] Merge the [agents fix PR](https://github.com/nomad-xyz/rust/pull/110) (@ltchang2019, @anna-carroll)

### Staging
 - [x] Deploy keymaster to staging (@arnaud036)
 - [x] Deploy agents to full staging setup (@arnaud036, @ltchang2019 )
 - [ ] Deploy indexer against staging (@arnaud036)
 - Fix staging indexer (erroring out with same message as dev indexer)
 - [x] Notify Connext that staging is ready (@ltchang2019) 
 - [ ] Deploy staging GUI (@ErinHales)
    - [ ] Fix GUI status tracking see [comment](https://github.com/nomad-xyz/monorepo/discussions/223#discussioncomment-2612014)
 - [x]  QA token transfer E2E (@ErinHales)
    - [x] check sending tokens 
    - [x] check that tokens are received 

### Release Staging
 - [ ] Update docs site with new domains (staging domains, probably should not add dev domains in docs)
 - [ ] Setup testnet tokens
    - [ ] define which token(s) are needed to be pre-bridged for testoooors
    - [ ] transfer tokens
    - [ ] set details
    - [ ] re-index in etherscan 
- [ ] Notify other external partners that staging is ready 
   - [ ] probably inc. dev-announcements in Discord

**Comment by anna-carroll** [comment]
are we set up to deploy the GUI for both dev and staging environments?

**Reply by ErinHales** [reply]
PR open for bumping sdk. A few things needed to get staging deploy up but dev will be ready to go as soon as that's in.

**Comment by anna-carroll** [comment]
I QA'd the token transfer flow in dev and it worked smoothly, i [received tokens](https://rinkeby.etherscan.io/tx/0xcd4d9c75835258079fb26933270019da7216bc687832283af8f0dcc48f12060f) on the other end ✅

**Comment by anna-carroll** [comment]
@ltchang2019 [dev is flailing](https://rinkeby.etherscan.io/address/0x097a3eb6cd351ab28de67a0a0bda9e0e32733da7) with `!proven` on the Replica - is this due to the latest code not being deployed to dev?

**Reply by anna-carroll** [reply]
also have `!MessageStatus.None` [here](https://rinkeby.etherscan.io/tx/0x388d32064ff0cf50fcb1a8e609b284e0e1266f0b8e37d3a19db13c380603387e)

**Comment by luketchang** [comment]
@ErinHales 

Hmm it looks like the GUI is saying tx from kovan --> rinkeby is [stuck in the dispatch phase](https://kovan.etherscan.io//tx/0x7dbc16c06740ae444b8d5473226f1a7f4e079d893d3ef284dc94223ec2217d76).

However I received tokens on the other side [here](https://rinkeby.etherscan.io/tx/0xd774281da6d79fa8b74d9c6d8668d58bd14eef5f4508a6c57e45cf315ddcdf5c)
